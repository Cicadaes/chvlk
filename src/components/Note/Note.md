Note documentation here...

```
<Note hideActionName="Help" showActionName="Help">
  <p>Regular payments are processed twice a month for all affiliates:</p>
  <ul>
    <li>During the first 3 business days of the month;</li>
    <li>Within 3 business days after the 15th of each month.</li>
  </ul>

  <p>At least $10 on your account at the start of regular payments:</p>
  <ul>
    <ol>At least $10 on your account at the start of regular payments;</ol>
    <ol>At least 5 active traders over your time in the affiliate program;</ol>
    <ol>A confirmed payment method;</ol>
  </ul>
</Note>
```