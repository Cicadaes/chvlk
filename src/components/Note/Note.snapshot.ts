import processSnapshots from '../../../utils/snapshots';
import Note, { INoteProps } from './Note';

const snapshots = [
  {
    props: {
      children: 'test',
      hideActionName: 'Hide help',
      showActionName: 'Show help',
    },
  },
];

processSnapshots<INoteProps>(Note, snapshots);
