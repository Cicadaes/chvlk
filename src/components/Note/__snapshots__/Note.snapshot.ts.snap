// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Note snapshots 1`] = `
<div>
  <styled.div>
    <styled.div
      onClick={[Function]}
    >
      <styled.info.svg />
      <styled.span>
        Hide help
      </styled.span>
      <styled.triangle.svg
        isOpen={true}
      />
    </styled.div>
  </styled.div>
  <styled.div
    isOpen={true}
  >
    test
  </styled.div>
</div>
`;
